Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/SN3A-Memory-Consumption/memory-consumption

```diff
@@ -0,0 +1,177 @@
+EOF
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-provision-mem-check-jiva metadata="")
+echo $test_name
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+
+cp apps/busybox/deployers/run_litmus_test.yml busybox_memcheck_jiva.yml
+
+sed -i -e 's/app: busybox-litmus/app: mem-busybox-jiva/g' \
+-e 's/app=busybox-sts/app=busybox-mem-jiva/g' \
+-e 's/value: statefulset/value: deployment/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
+-e 's/value: app-busybox-ns/value: mem-check-jiva/g' busybox_memcheck_jiva.yml
+
+cat busybox_memcheck_jiva.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:mem-busybox-jiva' job=busybox_memcheck.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:memory-consumption-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id SN3A 3-functional "Check if the memory consumption doesn't cross the desired threshold for known workload" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "*******Checking Memory Consumption**********"
+
+run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=memcheck-test metadata=${run_id})
+echo $test_name
+cd litmus
+
+################
+# LitmusBook 2 #
+################
+
+cp apps/prometheus/functional/memcheck/run_litmus_test.yml memcheck_test_jiva.yml
+
+
+: << EOF
+  --------------------------------------------------------------------------------------------------------------------     
+ | specAttribute     | kind   |         default value               | test specifc value                              |
+  ------------------------------------------------------------------------------------------------------------------- |
+ | appLabel          | env    | name=prometheus                     | app=busybox-mem-jiva                            |
+ | pvcName           | env    | prometheus-claim                    | openebs-busybox                                 | 
+ | storage clas      | env    | openebs-cstor-sparse                | openebs-jiva-default                            |
+ | Action            | env    | value: provision                    | value: deprovision                              |
+ | Litmus Job name   | name   | generateName: litmus-busybox-deploy | generateName: busybox-mem-check-deprovision-jiva|
+ | Litmus job label  | label  | app: memcheck-cstor-test            | memcheck-cstor-test-jiva                        |
+  ---------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/name=prometheus/app=busybox-mem-jiva/g' \
+-e 's/prometheus-claim/openebs-busybox/g' \
+-e 's/app: memcheck-cstor-test/app: memcheck-cstor-test-jiva/g' \
```

Comment:
@harshita-sharma-mayadata this mem-check test is for jiva can you avoid using cStor in the job labels?